lib/matplotlib/axes/_base.py

```diff
@@ -1390,8 +1451,26 @@ def get_data_ratio_log(self):
 
     def apply_aspect(self, position=None):
         """
-        Use :meth:`_aspect` and :meth:`_adjustable` to modify the
-        axes box or the view limits.
+        Adjust the axes so that it fulfills its aspect setting.
+
+        Depending on `.get_adjustable` and `.get_anchor` this will either modify
+        they axes box or the view limits.
```

Comment:
"...the Axes" (I think we capitalize Axes, maybe incorrect, but I think it makes sense since its not the plural of axis here?)

Reply:
We capitalize `Axes` because it's a Python class and that's PEP8 style. We don't capitalize axes if it's not referring to the class/an instance.

Reply:
I think we are referring to an instance here...

Reply:
Changed this and a few other cases in the above functions.

Note that the distinction is by far not always made in the code. Speaking just of `_AxesBase`, maybe 2/3 the cases use "axes" where it should be "Axes" according to your rules. IMO this distinction is not too important in many cases, unless you want a strong connotation to the class-type, in which case you would use the reference `` `Axes` `` anyway.

Reply:
Yeah, I only suggest it because of the linguistic strangeness of referring to "axes" as singular, so at least if we use "Axes" we can say its a singular proper noun.